src/components/radio/StyledIcon.ts

```diff
@@ -0,0 +1,38 @@
+import { styled } from 'emotion-solid'
+import tw from 'twin.macro'
+
+import type { IRadioProps } from './Radio'
+
+type styledRadioProps = IRadioProps & JSX.IntrinsicElements['span']
```

Comment:
This should be capitalised:
`StyledRadioProps = RadioProps & JSX.IntrinsicElements['span']`